python/tests/test-5-operator.py

```diff
@@ -23,7 +23,7 @@
 import check
 import buildmats as bm
 
-TOL = np.finfo(float).eps * 256
+TOL = libceed.lib.CEED_EPSILON * 256
```

Comment:
I think we should add this to `constants.py` so users have `libceed.EPSILON`

Reply:
This was added in 6cd68c5894ee60cd26737e0b42d38c37386d6369